lib/animina_web/live/profile_live.ex

```diff
@@ -307,20 +305,16 @@ defmodule AniminaWeb.ProfileLive do
           id: user_flag.flag.id,
           name: get_translation(user_flag.flag.flag_translations, language),
           emoji: user_flag.flag.emoji
-        },
-        category: %{
-          id: user_flag.flag.category.id,
-          name: get_translation(user_flag.flag.category.category_translations, language)
         }
       }
     end)
   end
 
   defp fetch_stories_and_flags(user, language) do
-    stories = fetch_stories(user.id)
+    stories = user.stories
 
     flags =
-      fetch_flags(user.id, :white, language)
+      fetch_flags(user, :white, language)
```

Comment:
I have the feeling that `fetch` is a misleading wording here. For me this feels more like a `filter`. Same with other instances in this PR.

Reply:
Yeah , Let me change that for better readability